assets/js/admin-ga-settings.js

```diff
@@ -0,0 +1,41 @@
+jQuery(document).ready( function($) {
+
+	var ecCheckbox        = $( '#woocommerce_google_analytics_ga_enhanced_ecommerce_tracking_enabled' );
+	var uaCheckbox        = $( '#woocommerce_google_analytics_ga_use_universal_analytics' );
+	var gtagCheckbox      = $( '#woocommerce_google_analytics_ga_gtag_enabled' );
+
+	updateToggles();
+
+	ecCheckbox.change(updateToggles);
+	uaCheckbox.change(updateToggles);
+	gtagCheckbox.change(updateToggles);
+
+	function updateToggles() {
+		var isEnhancedEcommerce  = ( true === ecCheckbox.is( ':checked' ) );
+		var isUniversalAnalytics = ( true === uaCheckbox.is( ':checked' ) );
+		var isGtag               = ( true === gtagCheckbox.is( ':checked' ) );
```

Comment:
Why do we need `true === `? We can just do `var isGtag = gtagCheckbox.is( ':checked' );`, no? 🤔

Reply:
Good spot → c57e2ddc8901afe13ae7120f7ec3787d6f34c5dc